charmhelpers/contrib/openstack/context.py

```diff
@@ -800,6 +801,8 @@ def configure_cert(self, cn=None):
         write_file(path=os.path.join(ssl_dir, key_filename),
                    content=b64decode(key))
 
+        service_restart('apache2')
```

Comment:
Any reason not to do a service_reload here instead?

This will in effect call apachectl graceful and issuing a reload without impacting any on-going connections.